WooCommerce/Classes/Tools/InfiniteScroll/ScrollWatcher.swift

```diff
@@ -0,0 +1,43 @@
+import UIKit
+
+/// Watches for scroll content offset changes and notifies its subscribers if the scroll position is over a threshold.
+///
+final class ScrollWatcher {
+    /// Emits a stream of scroll position percentages from 0.0 (the beginning) to 1.0 (the end) if over a given threshold whenever the user scrolls a
+    /// `UIScrollView` or subclass.
+    var scrollTrigger: Observable<Double> {
+        scrollTriggerSubject
+    }
+    private let scrollTriggerSubject: PublishSubject<Double>
+    private let scrollPositionThreshold: Double
```

Comment:
I think we can leave off the `scroll*` in the names of these properties. I think they're obvious because the class has `Scroll*` too. 🤔

Reply:
makes sense! updated in https://github.com/woocommerce/woocommerce-ios/pull/2747/commits/2f42e68a3b2bbbec9a582a2341dbbcfbc15f9e6d